.claude/commands/triage-feedback.md

```
@@ -0,0 +1,244 @@
+
+## Required MCPs
+
+Before doing anything, verify these MCPs are connected by attempting a discovery call:
+
+- `mcp__claude_ai_PostHog__exec` (PostHog hosted MCP)
+- `mcp__supabase-prod__execute_sql` (Supabase prod MCP)
+- `mcp__grafana__list_datasources` or similar (Grafana MCP)
+
+If any are missing, print:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)